src/cli/src/export.rs

```diff
@@ -300,19 +366,47 @@ impl Export {
         let timer = Instant::now();
         let db_names = self.get_db_names().await?;
         let db_count = db_names.len();
+        let s3_operator = if self.s3 {
+            Some(self.build_s3_operator().await?)
+        } else {
+            None
+        };
+
```

Comment:
Can we use the opendal operator for both S3 and FS?

Reply:
of course will try and fix them

Reply:
Done